### ISSUE TITLE
Dimensional Transceivers ?? [EnderIO]

### ISSUE THREAD

**Issue by Ealendir** [RS485/LogisticsPipes#568]
is it possible to connect two Logistic Pipes Segments over Dimensional Transceivers from EnderIO ? I thought the API was implemented at some point...
it does not seem to work however.....

**Comment by loordgek** [comment]
what is your setup?

**Comment by davboecki** [comment]
IIRC we only support the hypercube. We don't support the Dimensional Transceivers anymore.

**Comment by Xuerian** [comment]
Hypercube is the old one, Dimensional Transceiver is the new one: https://github.com/RS485/LogisticsPipes/pull/511 

That said I am having trouble getting it to work now as well: https://a.pomf.se/kzarol.webm

~~Worked fine in 1.6.4/TPPI1~~

**Comment by Xuerian** [comment]
I'm bad at github, apparently. It's somewhere in 0.9.

https://github.com/RS485/LogisticsPipes/commit/630c468dc7b4393bdf027757f82d0a432debf726#diff-6f081b53eef8f80778a5ee0e28b8ae2f

.. I think.

**Comment by bziemons** [comment]
@Xuerian what are those batteries and do they have an item input slot?

**Comment by Xuerian** [comment]
@theZorro266 Those are Creative Capacitor Banks from EnderIO. They do have an inventory.

I tested with only Creative Energy Cells, next to a similar pair of tesseracts: http://i.imgur.com/GE39vYO.jpg (Not sure if you wanted it tested, but there you go)

**Comment by davboecki** [comment]
That's probably your problem. There can't be any TileEntity next to the tesseract/transiver iirc. LP denies the connection when there is one.

**Comment by Xuerian** [comment]
@davboecki I double checked for you, that doesn't seem to be the problem. http://i.imgur.com/1dwsvUM.jpg

See:
https://github.com/RS485/LogisticsPipes/search?utf8=%E2%9C%93&q=hypercube
https://github.com/RS485/LogisticsPipes/search?utf8=%E2%9C%93&q=transciever

Vs https://github.com/RS485/LogisticsPipes/commit/630c468dc7b4393bdf027757f82d0a432debf726#diff-6f081b53eef8f80778a5ee0e28b8ae2f (On 0.9)
and https://github.com/RS485/LogisticsPipes/commit/ec8119bb43c97ec83932a39908e0238cad219b1f (on 0.8)

**Comment by davboecki** [comment]
Pipes are TileEntitys as well.

**Comment by AartBluestoke** [comment]
Would it be possible to, print a message to all people on the chunk the
first time a connection to a tesseract is rejected due to conflicting
destinations?

First=only once per person per login, ie a client side flag for of they
have seen the message.

Alternatively we could work with enderio to gain 'exclusive output' from
the tesseract, rather than it round-robin, or random.

**Comment by bziemons** [comment]
Sorry, there was quite a misunderstanding. I spoke with @davboecki seconds ago..
The Hypercube is "deprecated" as Prototik in #511 mentioned and yes the Dimensional Transceivers are new. The commit from Prototik was manually merged into the 0.9 branch though, because we didn't want to call it stable at that time.

So the tl;dr: Dimensional Transceivers are only supported in 0.9 and you need to get the 0.9 version to use them with LP.
